force_wfmanager/tests/test_wfmanager.py

```diff
@@ -138,14 +138,14 @@ def test_result_task_exit(self):
             for window in self.wfmanager.windows:
                 window.close = mock.Mock(return_value=True)
             self.review_task.exit()
-            self.assertTrue(self.review_task.window.close.called)
+            self.assertTrue(self.wfmanager.exit)
```

Comment:
The tests passed, but I got some funny message:

`KeyError: <weakref at 0x0000023A93E84EF8; to 'TasksPlugin' at 0x0000023A93CAD938>`

Reply:
This is failing on the Travis build, so we'll need to work out what's going on here. I think we can do this in a new PR, since it seems like it will need a bit of attention.